documentation/docs/user_guide/obsolete_and_deprecated_features/deprecated_features.md

```diff
@@ -0,0 +1,37 @@
+# Deprecated features for CABLE users #
+
+Usage of the deprecated user features described here is discouraged as they are scheduled to become obsolete
+and removed from future versions of CABLE. Attempts to use such features in runs of current versions of CABLE
+should result in general in a warning message.
+
+Related content which might be of interest:
+
+  - [Obsolete user features](./obsolete_features.md)
+  - [Deprecated developer features](../../developer_guide/obsolete_and_deprecated_features/deprecated_features.md)
+  - [Obsolete developer features](../../developer_guide/obsolete_and_deprecated_features/obsolete_features.md)
+  
+The template below can be used to report deprecated user features (one feature per block):
+
+## List of deprecated user features ##
```

Comment:
Same comment about the format to list a feature.

Reply:
I have adopted your format